resources/rules/post_identifier_merge/temp/abstraction/gene_or_gene_product_abstraction.clj

```diff
@@ -14,7 +11,7 @@
 
          (?/has_indirect_template_restriction owl/someValuesFrom ?/gene)
          (?/has_indirect_template_restriction owl/onProperty obo/pr#has_gene_template) ; PR:has_gene_template
-         (?/gene_product rdfs/subClassOf?/has_indirect_template_restriction))
+         (?/gene_product rdfs/subClassOf? /has_indirect_template_restriction))
```

Comment:
Nice catch.  The space needs to move one to the left so that it looks like ?/has_indirect_template_restriction